src/main/java/lux/fm/bookingservice/model/Payment.java

```diff
@@ -0,0 +1,43 @@
+package lux.fm.bookingservice.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import java.net.URL;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "payments")
+public class Payment {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    @Enumerated(EnumType.STRING)
+    private PaymentStatus status;
+
+    @OneToOne
+    @JoinColumn(name = "booking_id")
+    private Booking booking;
+
+    @Column(nullable = false)
+    private URL sessionUrl;

```

Comment:
I am not sure that this type will be valid in the future, so replace it by String